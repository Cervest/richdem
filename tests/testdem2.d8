ncols         10
nrows         10
xllcorner     421568
yllcorner     4872699
cellsize      3
NODATA_value  -1
3 3 3 3 3 3 3 3 3 3
3 3 3 3 3 3 3 3 3 3
3 3 3 3 3 3 3 3 3 3
3 3 3 3 3 3 3 3 3 3
3 3 3 3 3 3 3 3 3 3
4 3 2 3 4 3 2 3 4 3
3 3 3 3 3 3 3 3 3 3
3 3 3 3 3 3 3 3 3 3
3 3 3 3 3 3 3 3 3 3
3 3 3 3 3 3 3 3 3 3