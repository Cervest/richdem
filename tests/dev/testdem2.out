  5  20   5  10   5  20   5  10   5  15
  4  19   4   9   4  19   4   9   4  14
  3  18   3   8   3  18   3   8   3  13
  2  17   2   7   2  17   2   7   2  12
  1  16   1   6   1  16   1   6   1  11
  5   5   5   5   5   5   5   5   5   5
  4   4   4   4   4   4   4   4   4   4
  3   3   3   3   3   3   3   3   3   3
  2   2   2   2   2   2   2   2   2   2
  1   1   1   1   1   1   1   1   1   1
